packages/contractkit/src/wrappers/BaseWrapper.ts

```diff
@@ -20,6 +20,22 @@ export abstract class BaseWrapper<T extends Contract> {
     // TODO fix typings
     return (this.contract as any)._address
   }
+
+  /** Contract getPastEvents */
+  getPastEvents(
```

Comment:
not sure about making it private.. maybe protected.

Reply:
Also.... you might want to warn people about doing a query too far in the past or too big. Geth won't like...

cc @asaj @kevjue